src/Entity/Model/Bruger.php

```diff
@@ -79,6 +79,7 @@ class Bruger
     #[Groups(['bruger:item'])]
     private string $lokation;
 
+    // Private constructor to avoid instantiation as entity represents a database view.
```

Comment:
Actually, a MySQL view can be updatable. The constructor is private because we're creating a read-only entity (cf. line 16).

Reply:
Updated comments.